internal/locate/region_request.go

```diff
@@ -616,7 +616,8 @@ func (state *accessFollower) next(bo *retry.Backoffer, selector *replicaSelector
 	if selector.targetIdx < 0 {
 		leader := selector.replicas[state.leaderIdx]
 		leaderEpochStale := leader.isEpochStale()
-		leaderInvalid := leaderEpochStale || state.IsLeaderExhausted(leader)
+		leaderUnreachable := leader.store.getLivenessState() != reachable
```

Comment:
If liveness state is unknown, should it be treated as invalid?

Reply:
I'm not sure actually, I just see `accessKnownLeader` will fallback to `tryFollower` when `liveness != reachable`.